call-for-testing/action.yaml

```diff
@@ -34,6 +34,23 @@ inputs:
   store-token:
     description: "A token with permissions to upload to the specified channel"
     required: false
+  testing-instructions:
+    description: "Custom instructions for testing the `candidate` snap"
+    default: |
+      1. Stop the application if it was already running
+      1. Upgrade to this version by running
```

Comment:
```suggestion
      2. Upgrade to this version by running
```

Reply:
Ah, this was deliberate -- the Markdown processor on Github will automatically ensure the numbers are all in the right order :)